circle.yml

```diff
@@ -59,12 +59,13 @@ jobs:
           name: Install Docker Compose
           command: |
             set -x
-            sudo pip install docker-compose
+            sudo pip install docker-compose>=1.18
       - run:
           name: Start container, verify it's running and start tests
           command: |
             set -x
             sudo sysctl -w vm.max_map_count=262144
+            docker-compose --version
```

Comment:
Hmm, this looks fine to me. It's really weird that we still get docker-compose 1.14 here. Maybe for the sake of trying merge the pip install line with this starting line?

Reply:
If you look at the changes before the most recent you'll see that I tried that as well.